### PR TITLE
Add info about query for files and wrap some lines

### PR DESCRIPTION
**Pull request by tmotyl**

### CODE REVIEW

File: docs/filelist.rst [hunk 1]
```diff
@@ -6,7 +6,9 @@ File listing made easy
 Get all files which matches the query
 -------------------------------------
 
-Create `GoogleDriveFileList`_ instance with `parameters of Files.list()`_ as ``dict``. Call `GetList()`_ and you will get all files that matches your query as a list of `GoogleDriveFile`_.
+Create `GoogleDriveFileList`_ instance with `parameters of Files.list()`_ as ``dict``. 
+Call `GetList()`_ and you will get all files that matches your query as a list of `GoogleDriveFile`_.
+The syntax and possible option of the query ('q') parameter can be found in `search for files` Google documentation.
```
Comment (shcheklein) [inline docs/filelist.rst:11]: should it be ``q``?

Comment (shcheklein) [reply docs/filelist.rst:11]: I mean backquotes or double backquotes to highlight it properly

File: docs/filelist.rst [hunk 2]
```diff
@@ -24,7 +26,9 @@ You can update metadata or content of these `GoogleDriveFile`_ instances if you
 Paginate and iterate through files
 ----------------------------------
 
-*PyDrive* provides Pythonic way of paginating and iterating through list of files. All you have to do is to limit number of results with ``maxResults`` parameter and build ``for`` loop retrieving file list from API each iteration.
+*PyDrive* provides Pythonic way of paginating and iterating through list of files. 
+All you have to do is to limit number of results with ``maxResults`` parameter and build ``for`` loop retrieving
```
Comment (shcheklein) [inline docs/filelist.rst:30]: Let's change the style a bit, please. Something like this should be better:


Here is an example how to do this, ``maxResults`` below defines how many files it retrieves at once and we wrap it into a `for` loop to iterate:

**Comment by tmotyl** [comment]
@shcheklein I've fixed the wording and resolved conflict

**Review by shcheklein — APPROVED** [review]

**Comment by shcheklein** [comment]
thanks @tmotyl !

**Comment by shcheklein** [comment]
I'll release the new version of docs when/if we merge that #148

**Pull request merged**